apps/condo/domains/organization/schema/OrganizationEmployee.test.js

```diff
@@ -421,7 +458,11 @@ describe('OrganizationEmployee', () => {
         const [role] = await createTestOrganizationEmployeeRole(client1, client1.organization)
 
         // client1 invite client2 by phone with wrong name!
-        const [invite2_0] = await inviteNewOrganizationEmployee(client1, client1.organization, { phone: client2.userAttrs.phone, name: 'TEST1', email }, role)
+        const [invite2_0] = await inviteNewOrganizationEmployee(client1, client1.organization, {
+            phone: client2.userAttrs.phone,
+            name: 'TEST1',
```

Comment:
generate random pls

Reply:
Its not my, in other test use similar names